### PR TITLE
KLS-1659: Change style of LTE module cards

### PR DESCRIPTION
**Pull request by LuptonM**
(Please add a sentence or two explaining the context of this PR. Reviewers can go to the ticket for detail.)
CONTEXT: This changeset adds/removes/updates [summary of feature/area/code] so that [summary of benefit].

_Tick or delete as appropriate:_

### Workflow

- [ ] Ticket exists in Jira https://uktrade.atlassian.net/jira/software/projects/KLS/boards/359?assignee=63343adf88ed2ebef97c64ac&isInsightsOpen=true&selectedIssue=KLS-1659
- [ ] Jira ticket has the correct status.
- [ ] A clear/description pull request messaged added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.
- [ ] I have checked that my PR is using the latest package versions of: great-components, directory-constants, directory-healthcheck, directory-validators, directory-components, directory-api-client, directory-ch-client, django-staff-sso-client, directory-forms-api-client, directory-sso-api-client, sigauth

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/templatetags/content_tags.py [hunk 1]
```diff
@@ -14,14 +14,22 @@
...
+
+# from core.helpers import millify
 from core.models import DetailPage, LessonPlaceholderPage, TopicPage
 
 logger = logging.getLogger(__name__)
 
 register = template.Library()
 
 
+def millify(n):
```
Comment (divyaparameswaran) [inline core/templatetags/content_tags.py:26]: Any reason we moved this func here instead of the helpers ?

Comment (LuptonM) [reply core/templatetags/content_tags.py]: i had moved it as i put some logic into a page helper and was havign circular issues but dont need it now

File: core/templatetags/content_tags.py [hunk 2]
```diff
@@ -12,9 +12,10 @@
 from django.utils.html import format_html
 from django.utils.http import urlencode
 from django.utils.safestring import mark_safe
-
-from core.constants import BACKLINK_QUERYSTRING_NAME
 from core.helpers import millify
+from core.constants import BACKLINK_QUERYSTRING_NAME
+
+# from core.helpers import millify
```
Comment (divyaparameswaran) [inline core/templatetags/content_tags.py:18]: Don't think we need this line as the import is on line 15 .

**Review by divyaparameswaran — APPROVED** [review]
lgtm

**Review by lewis-coulson-dit — CHANGES_REQUESTED** [review]
You need to use the card component not just adding the "great-card" class to this

**Review by divyaparameswaran — APPROVED** [review]

File: core/templatetags/content_tags.py [hunk 3]
```diff
@@ -418,3 +419,33 @@ def get_icon_path(url):
...
         return ''
+
+
+@register.simple_tag
+def render_curated_topic_card_content(page, completed_lessons):
+    if completed_lessons is None or not hasattr(completed_lessons, '__iter__'):
+        completed_lessons = []
+
+    if str(page.id) in map(str, completed_lessons):
+        html_content = f"""
```
Comment (hal274) [inline core/templatetags/content_tags.py:430]: @LuptonM any plans to address codecov?

**Review by hal274 — APPROVED** [review]

**Comment by hal274** [comment]
Merging as the outstanding comment from @lewis-coulson-dit has been addressed and he is off work atm

**Pull request merged**